src/main/java/com/dynatrace/jenkins/dashboard/TAReportingBuildAction_2_0_0.java

```diff
@@ -32,27 +32,31 @@
 import com.dynatrace.jenkins.dashboard.model_2_0_0.TAReport;
 import com.dynatrace.jenkins.dashboard.rest.ServerRestURIManager;
 import com.dynatrace.jenkins.dashboard.utils.Utils;
-import hudson.model.AbstractBuild;
 import hudson.model.Action;
+import hudson.model.Run;
 import jenkins.model.GlobalConfiguration;
+import jenkins.model.RunAction2;
+import jenkins.tasks.SimpleBuildStep;
 import org.kohsuke.stapler.StaplerProxy;
 
 import java.net.URISyntaxException;
 import java.text.MessageFormat;
+import java.util.Collection;
+import java.util.Collections;
 
 /**
  * Created by krzysztof.necel on 2016-02-05.
  */
-public class TAReportingBuildAction_2_0_0 implements Action, StaplerProxy {
+public class TAReportingBuildAction_2_0_0 implements Action, StaplerProxy, SimpleBuildStep.LastBuildAction, RunAction2{
```

Comment:
check your formatter, opening brackets should be preceded with a space
fix all the occurences